### DISCUSSION TITLE
Relative links are not working

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [community/community#67750] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

According to the documentation: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

We should be able to use a link like `[something](docs/something.md)`

Unfortunately this does not work (sometimes).

Given this repo: https://github.com/LizardByte/ThemerrDB

There are two urls, both pointing to files in the same folder.

- `[Theme Guidelines](docs/Theme_Guidelines.md)`
- `[YouTube Share](docs/YouTube_Share.md)`

The first one works, the second one doesn't (when on the main repo page).

But they both work, if at this page: https://github.com/LizardByte/ThemerrDB/blob/master/README.md

**Comment by SenaJp** [comment]
Remove the `>` on line:

` > See [YouTube Share](docs/YouTube_Share.md) for further instruction.`

**Reply by ReenigneArcher** [reply]
That probably is the cause... but it should still work within a quote block.

**Reply by SenaJp** [reply]
Did you get to test it out of the quote block?

**Reply by Maetveis** [reply]
I have the same issue, but with lists. See https://github.com/Maetveis/rocm-examples/tree/fix_broken_links

Moving the items out of the lists does fix it, but it really should work with them too.

**Reply by arunsathiya** [reply]
Were relative links never supported on quote blocks?

Particularly surprising that it works on the actual README file path, but not on the homepage of a project. Example: `src/calculateRank.js` does not work on https://github.com/anuraghazra/github-readme-stats but works on https://github.com/anuraghazra/github-readme-stats/blob/master/readme.md

**Reply by ReenigneArcher** [reply]
@arunsathiya I experienced the same... everything worked when navigating to the specific file, but didn't work on the repo home page.

I ended up taking the link out of the block. Not ideal, but I don't expect it to get fixed any time soon.

**Reply by arunsathiya** [reply]
I submitted a support ticket as well, hopefully that would escalate this issue to the involved engineering teams!

**Reply by acusti** [reply]
@arunsathiya @ReenigneArcher this is definitely a bug, which i believe can be summarized as:

relative links are broken for the repo’s root readme file as rendered at the repo homepage if they are contained within a non-paragraph markdown block.

one other workaround i found is to update the link to have a leading slash and be built as from `github.com/*`, so the link in this case would be `/LizardByte/ThemerrDB/blob/master/docs/YouTube_Share.md`. this works in my testing (both at the root of the repo and in the file browser view) but might break if/when GitHub fixes this bug.

**Reply by AtmoFX** [reply]
That was driving me crazy... thank you for providing a workaround.

I do not see how the behavior where 2 identical links within the same file point to different addresses would make any sense.

**Comment by acusti** [comment]
other reports of this same bug:

https://github.com/orgs/community/discussions/70664
https://github.com/orgs/community/discussions/70735

**Comment by normanius** [comment]
Any chance this will get fixed soon? Relative path links are so useful in repos that are forked or otherwise duplicated. Furthermore, the chances are great that links can be handled locally by a Markdown viewer or IDE...

**Reply by ReenigneArcher** [reply]
I don't think anyone from GitHub has even seen this or acknowledged it... so probably won't be fixed anytime soon.

**Comment by lululeon** [comment]
Related:
https://github.com/github/markup/issues/1773
